### PR TITLE
Auto select test suite xccdf-id

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- TestSuite will default to first Benchmark in DataStream
- TestSuite offers non-interactive numeric selection of Benchmark

#### Rationale:

- Dealing with `xccdf-ids` is cumbersome and tedious.

### CODE REVIEW

File: tests/test_suite.py [hunk 1]
```diff
@@ -148,10 +155,34 @@ def get_logging_dir(options):
...
+
+        logging.info("Selected Benchmark is {0}".format(bench_number))
+        if bench_number < 0 or bench_number >= n_xccdf_ids:
+            msg = ("Please select a Benchmark number "
+                   "between 0 and {0}.".format(n_xccdf_ids-1))
+            raise RuntimeError(msg)
+        logging.info("To select a different Benchmark, "
+                     "use --xccdf-id-number option.")
+
+    return xccdf_ids[bench_number]
```
Comment (matusmarhefka) [inline tests/test_suite.py]: Won't this throw an exception when `bench_number >= 2` and `n_xccdf_ids <= 1`?

Comment (yuumasato) [reply tests/test_suite.py]: @matusmarhefka Should be handled now.

File: tests/test_suite.py [hunk 2]
```diff
@@ -148,10 +155,34 @@ def get_logging_dir(options):
...
 
 
+def auto_select_xccdf_id(datastream, bench_number):
+    xccdf_ids = xml_operations.get_all_xccdf_ids_in_datastream(datastream)
+    n_xccdf_ids = len(xccdf_ids)
+
+    if n_xccdf_ids > 1:
+        logging.info("The DataStream contains {0} "
+                     "Benchmarks".format(n_xccdf_ids))
+        for i in range(0, len(xccdf_ids)):
```
[the remark this answers is not in the document]
Comment (yuumasato) [reply tests/test_suite.py]: Ops, fixed

File: tests/test_suite.py (hunk 1 above)
Comment (matusmarhefka) [reply tests/test_suite.py]: What if `n_xccdf_ids = 1` and `bench_number` is 2 because I specified 2 using`--xccdf-id-number=2`?

Comment (yuumasato) [reply tests/test_suite.py]: I have moved handling of bad `bench_number` out of condition to print Benchmarks when multiple exist.

**Comment by matusmarhefka** [comment]
@yuumasato Nice patch, test suite command line is now much easier to use, thank you.

**Pull request merged**